spec/requests/api/v0/classified_listings_spec.rb

```diff
@@ -507,9 +522,37 @@ def put_classified_listing(id, **params)
       end
 
       it "cannot update another user listing" do
-        expect do
-          put_classified_listing(another_user_listing.id, title: "Test for a new title")
-        end.to raise_error(Pundit::NotAuthorizedError)
+        put_classified_listing(another_user_listing.id, title: "Test for a new title")
+        expect(response).to have_http_status(:unauthorized)
+      end
+
+      it "updates details if the listing has been bumped in the last 24 hours" do
+        listing.update!(bumped_at: 3.minutes.ago)
+
+        new_title = Faker::Book.title
+
+        put_classified_listing(listing.id, title: new_title)
+        expect(listing.reload.title).to eq(new_title)
+      end
+
+      it "does not update details if the listing hasn't been bumped in the last 24 hours" do
+        listing.update!(bumped_at: 24.hours.ago)
+
+        old_title = listing.title
+        new_title = Faker::Book.title
+
+        put_classified_listing(listing.id, title: new_title)
+        expect(listing.reload.title).to eq(old_title)
```

Comment:
since we're not really using `old_title` anywhere except for this line, might as well just do `expect(listing.reload.title).to eq(listing.title)`, right?

Reply:
will do, thanks!